src/main/java/net/neoforged/neoforge/items/VanillaInventoryCodeHooks.java

```diff
@@ -121,6 +124,21 @@ public static boolean insertHook(HopperBlockEntity hopper) {
                 .orElse(false);
     }
 
+    /**
+     * Added capability support for the Crafter dispensing the result
+     */
+    public static ItemStack insertCrafterOutput(Level level, BlockPos pos, CrafterBlockEntity crafterBlockEntity, ItemStack stack) {
+        FrontAndTop frontAndTop = level.getBlockState(pos).getValue(BlockStateProperties.ORIENTATION);
```

Comment:
This is causing a crash, because the block position parameter in this method is for the target block of the crafter, not the crafter itself. 

Either the `BlockPos#relative` call in the patch should be removed, or the `Direction` should be passed into this method and used in place of `frontAndTop.front()`.